ds_pract.-2_1906307178.c

```diff
@@ -0,0 +1,104 @@
+/*
+name:soham purohit
+enrollment no:1906307178
+class: 3 CE 3 
+data structure practical 2 (stack)
```

Comment:
Remove this un-necessary information about youself and write what the program will do